data/ignition/files/usr/local/share/assisted-service/pod.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
Did you just copy it from the assisted-service repo?

Reply:
[Yes](https://github.com/openshift-agent-team/fleeting/pull/2/commits/292e5e58f68acaf36ce831c09cc5aba8506f83fb).